github/handlers/ExecuteViewSubmitHandler.ts

```diff
@@ -481,7 +481,7 @@ export class ExecuteViewSubmitHandler {
                                         .getInteractionResponder()
                                         .openModalViewResponse(unauthorizedMessageModal);
                                 }else{                               
-                                    let issueComments = await getIssuesComments(this.http,repository,issueNumber);
+                                    let issueComments = await getIssuesComments(this.http,repository,accessToken.token,issueNumber);
```

Comment:
`let issueComments = await getIssuesComments(this.http,repository,accessToken?.token,issueNumber);`